source/intro_release_notes/release_notes/whats_new.rst

```diff
@@ -62,6 +62,10 @@ API & CLI
 - XSD files has been updated and completed
 - Pagination can be disabled using ``no-pager`` option.
 
+Storage
+--------------------------------------------------------------------------------
+- Free space of the KVM hypervisor is now updated faster for SSH and LVM transfer managers by sending HUP signal to collectd client.
```

Comment:
quick comment, what's new entries should have links to the documentation. This looks like a performance enhancement, so I'd like to the fs and lvm guides